WordPress/src/main/java/org/wordpress/android/ui/sitecreation/SiteCreationMainVM.kt

```diff
@@ -50,13 +59,28 @@ typealias NavigationTarget = WizardNavigationTarget<SiteCreationStep, SiteCreati
 class SiteCreationMainVM @Inject constructor(
     private val tracker: SiteCreationTracker,
     private val wizardManager: WizardManager<SiteCreationStep>,
-    private val siteNameABExperiment: SiteNameABExperiment
+    private val siteNameABExperiment: SiteNameABExperiment,
+    private val networkUtils: NetworkUtilsWrapper,
+    private val dispatcher: Dispatcher,
+    private val fetchHomePageLayoutsUseCase: FetchHomePageLayoutsUseCase
 ) : ViewModel() {
+    init {
+        // TODO: Remove the duplicate {,un}registration in the picker view model
```

Comment:
With the cancellation (interruption) flow, we need to have this in both places. The only thing left to remove is this comment, which I forgot to remove before merging :man_facepalming: .

Reply:
Removed here [5351d73](https://github.com/wordpress-mobile/WordPress-Android/pull/16619/commits/5351d7359ecba7942b946a72bb813cfe77ae1489).

Part of PR: #16619

Reply:
Thanks Ovi!